lib/mocha/configuration.rb

```diff
@@ -303,6 +306,17 @@ def reinstate_undocumented_behaviour_from_v1_9?
       @options[:reinstate_undocumented_behaviour_from_v1_9]
     end
 
+    # @private
+    def strict_keyword_argument_matching=(value)
+      raise "Strict keyword argument matching requires Ruby 2.7 and above." unless Mocha::RUBY_V27_PLUS
+      @options[:strict_keyword_argument_matching] = !!value
```

Comment:
Minor: I'd prefer to do without the double-bang here for now since it's inconsistent with the implementation of `#display_matching_invocations_on_failure?`. I'd be happy to consider changing the implementation of both of them separately.